zerver/outgoing_webhooks/generic.py

```diff
@@ -13,21 +13,19 @@ def __init__(self, base_url, token, user_profile, service_name):
         super(GenericOutgoingWebhookService, self).__init__(base_url, token, user_profile, service_name)
 
     def process_event(self, event):
-        # type: (Dict[str, Any]) -> Tuple[Dict[str ,Any], Dict[str, Any]]
+        # type: (Dict[str, Any]) -> Tuple[Dict[str ,Any], Any]
         rest_operation = {'method': 'POST',
                           'relative_url_path': '',
                           'base_url': event['base_url'],
                           'request_kwargs': {}}
         request_data = {"data": event['command'],
                         "message": event['message']}
-        return (rest_operation, request_data)
+        return rest_operation, json.dumps(request_data)
```

Comment:
What's the reason for having `json.dumps` here?  Can you please leave a comment?